.github/ISSUE_TEMPLATE/device-request.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Device Request
+about: This form helps in requesting a new device to be supported.
+title: "[DEVICE SUPPORT REQUEST] - Devicename"
+labels: Device Request
+assignees: ''
+
+---
+
+Before requesting a device, please make sure to search the open and closed issues for any requests in the past. Sometimes devices have been requested before but are not implemented yet due to various reasons. If there are no hits on your Product Number, please proceed.  
+
+**Device Name:**
+
+**Device Type:**
+- [ ] Light
+- [ ] Lock
+- [ ] Remote
+- [ ] Sensor
+- [ ] Siren
+- [ ] Thermostat
+- [ ] Other: 
+
+**Device Manufaturer:**
+
+**Device Product Number:**
```

Comment:
The difference between what's on the packaging and what deconz reports via basic cluster must become clearer. For integration, the latter is important only.

Reply:
What would you suggest?

Reply:
Either double it for packaging (this might just be nice to have unless the supported devices list is massively extended) and deconz GUI or something like "manufacturer/model identifier as per deconz GUI basic cluster"